test/lit/exec/strings.wast

```diff
@@ -414,6 +414,30 @@
     ;; Concatenating these surrogates creates '𐍈'.
     (string.concat (string.const "\ED\A0\80") (string.const "\ED\BD\88"))
   )
+
+  ;; CHECK:      [fuzz-exec] calling string.from_code_point
+  ;; CHECK-NEXT: [fuzz-exec] note result: string.from_code_point => string("A")
+  (func $string.from_code_point (export "string.from_code_point") (result stringref)
+    (string.from_code_point
+      (i32.const 65)
+    )
+  )
+
+  ;; CHECK:      [fuzz-exec] calling weird_code_point
+  ;; CHECK-NEXT: [fuzz-exec] note result: weird_code_point => string("\uffe8")
+  (func $weird_code_point (export "weird_code_point") (result stringref)
+    (string.from_code_point
+      (i32.const 1000)
+    )
+  )
+
+  ;; CHECK:      [fuzz-exec] calling invalid_code_point
+  ;; CHECK-NEXT: [trap invalid code point]
+  (func $invalid_code_point (export "invalid_code_point") (result stringref)
+    (string.from_code_point
+      (i32.const -83)
+    )
+  )
```

Comment:
Other test cases to add:

An isolated high surrogate: `i32.const 0xD800`
An isolated low surrogate: `i32.const 0xDC00`
A codepoint requiring a surrogate pair: `10348` (`𐍈`)

Reply:
Thanks, added. I also noticed a fuzz error on an unsigned case (that now works) that I also added.